<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>can_log_sender</class>
 <widget class="QWidget" name="can_log_sender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="set_file_pushButton">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_name_lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="circle_label">
       <property name="text">
        <string>循环次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="circle_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_circle_label">
       <property name="text">
        <string>当前次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="current_circle_lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="s_index_label">
         <property name="text">
          <string>数据起始索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bytes_index_label">
         <property name="text">
          <string>字节数索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="can_id_index_label">
         <property name="text">
          <string>CAN ID索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="permissable_can_id_label">
         <property name="text">
          <string>允许发送CAN ID列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="send_channel_label">
         <property name="text">
          <string>发送通道</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="send_delay_ms_label">
         <property name="text">
          <string>帧间延时ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="response_can_id_label">
         <property name="text">
          <string>特殊等待CAN ID列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="response_can_data_label">
         <property name="text">
          <string>特殊等待数据列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wait_can_data_index_label">
         <property name="text">
          <string>特殊等待数据索引</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wait_response_ms_label">
         <property name="text">
          <string>等待时间ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="s_index_lineEdit">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bytes_index_lineEdit">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="can_id_index_lineEdit">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="permissable_can_id_lineEdit">
         <property name="text">
          <string>154</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="send_channel_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="send_delay_ms_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="response_can_id_lineEdit">
         <property name="text">
          <string>254</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="response_can_data_lineEdit">
         <property name="text">
          <string>8D 81 8C 84</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wait_can_data_index_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wait_response_ms_lineEdit">
         <property name="text">
          <string>300</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="start_pushButton">
     <property name="text">
      <string>启动发送</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="send_progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
